### PR TITLE
add newFile.txt

### PR DESCRIPTION
**Pull request by LauraSchoenhals**
I made a file called newFile & I'm adding it into the new branch

### CODE REVIEW

**Review by zsk4 — APPROVED** [review]
This file is the best

**Review by jarbesfeld — APPROVED** [review]
great file!

**Review by iris-wang1 — COMMENTED** [review]
looks amazing!!

**Review by iris-wang1 — CHANGES_REQUESTED** [review]
bad, please add more content.

File: newFile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hey_a_new_thing
```
Comment (zsk4) [inline newFile.txt]: Use CamelCase plz

**Review by zsk4 — CHANGES_REQUESTED** [review]
See line 1 comment

File: newFile.txt (hunk 1 above)
Comment (LauraSchoenhals) [reply newFile.txt]: fine

**Review by iris-wang1 — APPROVED** [review]

**Review by zsk4 — APPROVED** [review]
Looks Good

**Pull request merged**